2017/kickstart.md

```diff
@@ -0,0 +1,42 @@
+---
+layout: news
+title: 2017 Kickstart
+permalink: /2017/kickstart
+date: 2016-10-01
+---
+
+Kickstart is the event which kicks off a competition year, where the game will be announced, and robotics kits will be handed out.
+
+The venues for Student Robotics 2017 kickstart are:
+
+- University of Southampton
+- University of Bristol
+- The Kings Centre, Oxford
+
+Teams are welcome to attend any venue they wish. However; they should inform their local team co-ordinators which venue they will attend, as we will need to arrange for the robotics kits to be at the venue they choose, and some venues have vehicle access restrictions which will need to be arranged beforehand.
```

Comment:
_How_ should they contact their LTCs? Have they been given contact details via another channel? Could we put a contact detail here to simplify the process?